media-gfx/qimgv/Manifest

```diff
@@ -0,0 +1,3 @@
+DIST qimgv-0.6.tar.gz 232586 SHA256 b4f5b5400e892ddcc1a417d8652eda25122b520308dd20fa9d9d5da4b5784892 SHA512 50156a6bdf08e8cc82d30ad23d1deef7f0c119c970c8e577d8b0de65f6b11827e0cf57593834ae7e8887dce5d83215cbd98e9c5b8905d6a3580943e432bdb6cb WHIRLPOOL 79bbc9325def7b84b31f383f0faa205623341e393e917dcb5dba5e8bf7376a0998cacb2cf20bc05a5bfe6acf8788fe7b3882864b1759962c0f00c7ab6c7e2ae4
+EBUILD qimgv-0.6-r1.ebuild 817 SHA256 744eae0a43b1d4a8f6f172494d0288b0a00c1080170945f67068bb749f76e17e SHA512 73cddb6343435be8113e143646ca65bacd80e79356417c33f089c4a40f611707f1ff8d77f1885b9048b60876b5b9479310e04ee8df986a3dc961fab0b831031e WHIRLPOOL a4d59f48d5e9046f12df355850055f5a75974ef88128f1cac7a0b27e5a4fbbd089215914582acc6aac869ad65a8e1b27aa24d0a23057a951d1216cc35224cc84
```

Comment:
For some reason, the repository you've cloned generates normal Manifests. Please set it up so that thin Manifests are generated. See: https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests